src/app/api/edge-config/route.ts

```diff
@@ -1,6 +1,6 @@
 import { NextResponse } from 'next/server';
 import { get } from '@vercel/edge-config';
-import { Category } from '../../../stores/edge-config-store';
+import { Category } from '../../../types/index';
```

Comment:
I *believe* index can be omitted. 
```suggestion
import { Category } from '../../../types/';
```

Reply:
(and it tends to be preferable, so all imports don't end with index)

Reply:
I have types/categories.ts that also has a category for the enum. unless I can combine them?

![image](https://github.com/user-attachments/assets/1d366009-38d0-48db-84c2-e08e58991089)

![image](https://github.com/user-attachments/assets/839f1597-b63d-4808-8d37-901a12907b20)